diff --git a/glue-codes/labview/src/WaveTank.f90 b/glue-codes/labview/src/WaveTank.f90
index d7edd11a2..7be10ee85 100644
--- a/glue-codes/labview/src/WaveTank.f90
+++ b/glue-codes/labview/src/WaveTank.f90
@@ -56,8 +56,6 @@ SUBROUTINE WaveTank_Init(   &
 !GCC$ ATTRIBUTES DLLEXPORT :: WaveTank_Init
 #endif
 
-IMPLICIT NONE
-
 CHARACTER(KIND=C_CHAR), INTENT(IN   ), TARGET :: MD_InputFile_c(IntfStrLen)
 CHARACTER(KIND=C_CHAR), INTENT(IN   ), TARGET :: SS_InputFile_c(IntfStrLen)
 CHARACTER(KIND=C_CHAR), INTENT(IN   ), TARGET :: AD_InputFile_c(IntfStrLen)
@@ -193,8 +191,6 @@ SUBROUTINE WaveTank_CalcOutput( &
 !GCC$ ATTRIBUTES DLLEXPORT :: WaveTank_CalcOutput
 #endif
 
-IMPLICIT NONE
-
 ! INTEGER(C_INT)                        :: delta_time
 INTEGER(C_INT)                        :: frame_number
 REAL(C_FLOAT),          INTENT(IN   ) :: positions_x(N_CAMERA_POINTS)
